pennylane/ops/qubit.py

```diff
@@ -237,7 +237,8 @@ class RX(Operation):
 
     * Number of wires: 1
     * Number of parameters: 1
-    * Gradient recipe: :math:`\frac{d}{d\phi}R_x(\phi) = \frac{1}{2}\left[R_x(\phi+\pi/2) - R_x(\phi-\pi/2)\right]`
+    * Gradient recipe: :math:`\frac{d}{d\phi}f(R_x(\phi)) = \frac{1}{2}\left[f(R_x(\phi+\pi/2)) - f(R_x(\phi-\pi/2))\right]`
```

Comment:
We should update the recipes in the same manner for the controlled rotation gates too. Thanks!

Reply:
@johannesjmeyer you'll have to merge in latest master to see those gates (added in the meantime while this PR was waiting for final approval). Sorry for being slow!